docker-compose.yml

```diff
@@ -9,6 +9,15 @@ services:
       - "8080:8080"  # Adjust the port as needed
     restart: unless-stopped
 
+  Velocity_microservice:
```

Comment:
don't use underscore, java won't like it at all.
we will have to refer to this service with
http://Velocity_microservice:portNumber

java won't like that it has an underscore